ClothesApp/Infrastructure/Repositories/AddressRepository.cs

```diff
@@ -0,0 +1,57 @@
+using System.Linq.Expressions;
+using Application.Interfaces.Repositories;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Repositories;
+
+public class AddressRepository : IAddressRepository
+{
+    private readonly ClothesAppContext _dbContext;
+
+    public AddressRepository(ClothesAppContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public Task<Address> Insert(Address entity)
+    {
+        throw new NotImplementedException();
+    }
+
+    public async Task<Address> Update(Address address)
+    {
+        _dbContext.Addresses.Update(address);
+        await _dbContext.SaveChangesAsync();
+
+        return address;
+    }
+
+    public Task<IList<Address>> GetAll()
+    {
+        throw new NotImplementedException();
```

Comment:
This is worse than having the implementation, but not using it. Potentially this method can be helpful and it will not be obvious that it doesn't have implementation